addons/fluxcd/definitions/helm-release-def.yaml

```
@@ -57,6 +57,7 @@ spec:
         		name: context.name
         	}
         	spec: {
+        		timeout: parameter.installTimeout
```

```suggestion
        		timeout: parameter.timeout
```

When we talk about `helm`, we mean installing most of the time. So `timeout` can clearly express the install context.

---

Or it seems conflicts with the existing timeout.

---

There's conflict yeah.